generated/niswitch/session.py

```diff
@@ -164,7 +164,7 @@ class _SessionBase(object):
 
         var = session.channels[0,1].characteristic_impedance
     '''
-    continuous_scan = _attributes.AttributeViBoolean(1150002)
+    continuous_scan = _attributes.AttributeViBoolean(1250026)
```

Comment:
fixes should go in CHANGELOG

Reply:
It makes no difference to the client. Both IDs work and point to the same attribute.